hudi-bench/src/main/java/org/apache/hudi/bench/DeltaInputFormat.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.bench;
+
+/**
+ * Supported delta input data formats
+ */
+public enum DeltaInputFormat {
```

Comment:
can this simply be `InputDataFormat` ? Use of delta to denote input change log is not very apparent to me. better to avoid naming all these classes like that to avoid confusing with DeltaStreamer for e.g?

Reply:
I also find that this is expanding from testing correctness to testing if/how hudi handles differnt input format is unnecessary here?  IMO such format specific testing should be as a deltastreamer test. not here.

Reply:
@vinothchandar The last comments from on the PR were around using `delta` for naming : https://github.com/apache/incubator-hudi/pull/623#discussion_r276877339 hence used them.

Have had multiple conversations with @bvaradar on this as well on the scope (and some with you earlier). The idea was to be able to test all paths of DeltaStreamer so that as changes are made to it, it is hardened. Since deltastreamer supports different kinds of sources, the idea here is to be able to produce to these different sources so deltastreamer sources are also tested.